package.json

```diff
@@ -110,7 +110,7 @@
     ]
   },
   "engines": {
-    "node": ">=12.18.0 <15.x",
+    "node": ">=10.16.0 <15.x",
```

Comment:
We should all be on at least node [`v14`](https://nodejs.org/en/about/releases/). 10 is not longer maintained and 12 will reach EOL in April.